.github/workflows/main.yml

```diff
@@ -204,16 +204,16 @@ jobs:
           dart pub upgrade
           dart test
   test_flutter:
-    # strategy:
-    #   matrix:
-    #     version: ["3.16.0", "3.19.0", "3.21.0"]
+    strategy:
+      matrix:
+        version: [3.16.0, 3.19.0, 3.21.0]
```

Comment:
IMO supporting the latest stable version is good enough. I guess we have no choice but to support the older versions of packages pinned by Flutter stable, but going further back sounds annoying for relatively little value. If the latest drift release doesn't support the latest Flutter that's bad, but I'm fine with not supporting older Flutter versions.

Reply:
Sure